### PR TITLE
Move SC H4216 income tax reform to baseline

### LINKED ISSUE #7916: Implement SC H4216 income tax reform (2026+)
# Implement SC H4216 Income Tax Reform (2026+)

## Overview

South Carolina H.4216 was signed into law and takes effect for tax years beginning after 2025. This issue tracks moving the reform from contributed reform status to baseline implementation.

**Previous implementation**: PR #7494 implemented this as a contributed reform. This issue covers making it the default (baseline) for 2026+.

## Law Summary

SC H4216 is a comprehensive income tax reform that includes:

### 1. New Two-Bracket Tax Structure
| Bracket | Threshold | Rate |
|---------|-----------|------|
| 1 | $0 | 1.99% |
| 2 | $30,000 | 5.21% |

### 2. SC Income Adjusted Deduction (SCIAD) with Phaseout
Replaces federal standard/itemized deductions with a new state-specific deduction:

| Filing Status | Base Amount | Phase-out Start | Phase-out End |
|---------------|-------------|-----------------|---------------|
| Single | $15,000 | $40,000 | $95,000 |
| Married Filing Separately | $15,000 | $40,000 | $95,000 |
| Head of Household | $22,500 | $60,000 | $142,500 |
| Married Filing Jointly | $30,000 | $80,000 | $190,000 |
| Surviving Spouse | $30,000 | $80,000 | $190,000 |

Formula: `SCIAD = base_amount × (1 - (AGI - phase_out_start) / phase_out_width)`

### 3. EITC Capped at $200
The SC EITC (125% of federal) is capped at a maximum of $200.

## Status Checklist
- [ ] Parameters updated with 2026+ effective dates
- [ ] Variables updated to use new structure for 2026+
- [ ] Tests updated for baseline behavior
- [ ] CI passing
- [ ] PR ready for review

## References
- [Bill text](https://www.scstatehouse.gov/sess126_2025-2026/prever/4216_20260224.htm)
- [RFA Fiscal Impact Statement](https://www.scstatehouse.gov/sess126_2025-2026/fiscalimpactstatements/H4216%202026-01-13%20updated.pdf)
- Previous implementation: PR #7494, Issue #7493

---
*This issue serves as the central coordination point for moving SC H4216 to baseline.*

### PR DESCRIPTION
**Pull request by DTrim99**
## Summary

Moves the South Carolina H.4216 income tax reform from contributed reform status to baseline, making it the default for tax years beginning after 2025.

Closes #7916

## Regulatory Authority

- **Bill**: [SC H.4216 (Act No. 110)](https://www.scstatehouse.gov/sess126_2025-2026/bills/4216.htm)
- **Signed**: March 30, 2026
- **Effective**: Tax years beginning after 2025

### 1. New Tax Rates (Section 1 - Section 12-6-510(C))

| Taxable Income | Rate |
|---------------|------|
| $0 - $30,000 | 1.99% |
| Over $30,000 | 5.21% |

Note: The bill includes provisions for future automatic rate reductions (beginning 2027) based on revenue growth projections. These dynamic reductions are not modeled.

### 2. SC Income Adjusted Deduction - SCIAD (Section 3 - Section 12-6-1140(15))

Replaces federal standard/itemized deductions with state-specific deduction:

| Filing Status | Base Amount | Phase-out Start | Phase-out End |
|--------------|-------------|-----------------|---------------|
| Single | $15,000 | $40,000 | $95,000 |
| Married Filing Separately | $15,000 | $40,000 | $95,000 |
| Head of Household | $22,500 | $60,000 | $142,500 |
| Married Filing Jointly | $30,000 | $80,000 | $190,000 |
| Surviving Spouse | $30,000 | $80,000 | $190,000 |

Phase-out formula: `reduction = base_amount * (AGI - start) / width`, rounded down to nearest $10

### 3. EITC Cap (Section 7 - Section 12-6-3632)

The SC EITC (125% of federal EITC) is now capped at $200 maximum.

### 4. Taxable Income Calculation (Section 2 - Section 12-6-50)

For 2026+, SC does not adopt federal standard/itemized deductions. The calculation starts from federal AGI instead of federal taxable income.

## Files Added

**Parameters:**
- `parameters/gov/states/sc/tax/income/credits/eitc/max.yaml` - EITC cap
- `parameters/gov/states/sc/tax/income/deductions/sciad/amount.yaml` - SCIAD base amounts
- `parameters/gov/states/sc/tax/income/deductions/sciad/phase_out/start.yaml` - Phase-out thresholds
- `parameters/gov/states/sc/tax/income/deductions/sciad/phase_out/width.yaml` - Phase-out widths

**Variables:**
- `variables/gov/states/sc/tax/income/deductions/sciad/sc_sciad.py` - SCIAD calculation

**Tests:**
- `tests/policy/baseline/gov/states/sc/tax/income/sc_h4216_baseline.yaml` - Comprehensive test suite

## Files Modified

- `parameters/gov/states/sc/tax/income/rates.yaml` - Added 2026+ rate brackets
- `parameters/gov/states/sc/tax/income/subtractions/subtractions.yaml` - Added SCIAD to subtractions for 2026+
- `variables/gov/states/sc/tax/income/credits/eitc/sc_eitc.py` - Added $200 cap
- `variables/gov/states/sc/tax/income/sc_taxable_income.py` - Changed to start from AGI for 2026+

## Test Coverage

The test suite includes:
- EITC cap tests (at/above/below cap, pre-2026 no cap)
- SCIAD tests for all filing statuses
- SCIAD phase-out boundary tests
- Tax rate bracket tests
- Zero/negative income edge cases
- Non-SC resident tests

---

🤖 Generated with [Claude Code](https://claude.ai/code)

### CODE REVIEW

File: policyengine_us/variables/gov/states/sc/tax/income/sc_taxable_income.py [hunk 1]
```diff
@@ -8,10 +8,26 @@ class sc_taxable_income(Variable):
...
-        taxable_income = tax_unit("taxable_income", period)
-        additions = tax_unit("sc_additions", period)
+        # Check if we're in H4216 era (2026+)
+        # H4216 Section 2: SC does not adopt federal standard/itemized deductions
+        # H4216 Section 3: SC uses SCIAD instead, starting from AGI
+        year = period.start.year
         subtractions = tax_unit("sc_subtractions", period)
-        return max_(0, taxable_income + additions - subtractions)
+
+        if year >= 2026:
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/sc/tax/income/sc_taxable_income.py]: dont hard code the year, lets use an in_effect

File: policyengine_us/parameters/gov/states/sc/tax/income/deductions/sciad/phase_out/start.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+HEAD_OF_HOUSEHOLD:
+  0001-01-01: 0
+  2026-01-01: 60_000
+
+JOINT:
+  0001-01-01: 0
+  2026-01-01: 80_000
+
+SURVIVING_SPOUSE:
+  0001-01-01: 0
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/sc/tax/income/deductions/sciad/phase_out/start.yaml]: dont need 0s here

File: policyengine_us/variables/gov/states/sc/tax/income/deductions/sciad/sc_sciad.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        # fraction = (AGI - start) / width
+        excess_agi = max_(0, agi - phase_out_start)
+        phase_out_fraction = min_(1, excess_agi / phase_out_width)
+        # Calculate reduction
+        reduction = base_amount * phase_out_fraction
+        # Round reduction down to nearest $10
+        reduction_rounded = np.floor(reduction / 10) * 10
+        # Calculate deduction
+        deduction = max_(0, base_amount - reduction_rounded)
+        return deduction
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/sc/tax/income/deductions/sciad/sc_sciad.py]:
```suggestion
        return max_(0, base_amount - reduction_rounded)
```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Review by DTrim99 — COMMENTED** [review]
Findings:

- P3: `policyengine_us/parameters/gov/states/sc/tax/income/subtractions/subtractions.yaml:36` has trailing whitespace on `- sc_sciad`; `git diff --check upstream/main...HEAD` fails on this.
- P3: `changelog.d/sc-h4216-baseline.added.md:1` says `00 EITC cap`; it should read `$200 EITC cap` or `200 EITC cap`.

No blocking behavioral findings found. I verified the SCIAD reduction-rounding behavior against the enacted SC H.4216 text; the law rounds the reduction amount down to the next lower $10, matching the PR's current `sc_sciad` formula.

Verification:

- Passed: `py -3.11 -m policyengine_core.scripts.policyengine_command test policyengine_us/tests/policy/baseline/gov/states/sc/tax/income/sc_h4216_baseline.yaml -c policyengine_us` (`14 passed`)
- Initial attempt with the local `.venv` Python 3.14 failed before running tests because NumPy C extensions could not import (`DLL load failed while importing _multiarray_umath`).
